src/datepicker/datepicker.js

```diff
@@ -13,10 +13,33 @@ import Calendar from './calendar.js';
 import {getOverrides} from '../helpers/overrides.js';
 import getInterpolatedString from '../helpers/i18n-interpolation.js';
 import {LocaleContext} from '../locale/index.js';
-import {StyledInputWrapper} from './styled-components.js';
+import {StyledInputWrapper, StyledInputLabel} from './styled-components.js';
 import type {DatepickerPropsT} from './types.js';
 import DateHelpers from './utils/date-helpers.js';
 import dateFnsAdapter from './utils/date-fns-adapter.js';
+import type {LocaleT} from '../locale/types.js';
+
+export const DEFAULT_DATE_FORMAT = 'yyyy/MM/dd';
+
+const INPUT_DELIMITER = ' – ';
+
+const combineSeparatedInputs = (
```

Comment:
I'm concerned the purpose of this function might not be clear even after seeing it used in context. Let me know if that's the case and I'll leave a comment above it (I'm usually reluctant to do that because comments can get cluttery, and fall out of sync with the code itself).